src/components/BillCategoryTag/BillCategoryTag.story.svelte

```diff
@@ -7,8 +7,8 @@
 </script>
 
 <Hst.Story title="BillCategoryTag" layout={{ type: 'grid' }}>
-	<Hst.Variant title="BillCategory">
-		<BillCategoryTag {isLarge} />
+	<Hst.Variant title="PublicTransport">
+		<BillCategoryTag {isLarge} label="ขนส่งสาธารณะ" />
```

Comment:
I suggest adding the label to the control with `Hst.Text` as well, to make the prop visible to anyone who looking at this story. Then you can set the default value to `"ขนส่งสาธารณะ"`